Monika After Story/game/script-compliments.rpy

```diff
@@ -1358,3 +1358,21 @@ label mas_compliment_outfit_repeat:
         m 3hubsb "[other_quip]"
 
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_compliments_database,
+            eventlabel="mas_compliment_language",
+            prompt="Human language doesn't have a word for how much I love you",
```

Comment:
Personally, I think this can be a little shorter while retaining the message.

```suggestion
            prompt="No words can express how much I love you!",
```